<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>openMotor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/oMIconCycles.png</normaloff>resources/oMIconCycles.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutMotorEditor">
      <item>
       <widget class="MotorEditor" name="motorEditor" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>475</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>4</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutPropellant">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboBoxPropellant">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPropEditor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Propellant Editor</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetGrainList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Details</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutEditButtons">
        <item>
         <widget class="QPushButton" name="pushButtonMoveGrainUp">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>↑</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonMoveGrainDown">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>↓</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonEditGrain">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCopyGrain">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDeleteGrain">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutAddGrain">
        <item>
         <widget class="QComboBox" name="comboBoxGrainGeometry"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddGrain">
          <property name="text">
           <string>Add Grain</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutResults">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ResultsWidget" name="resultsWidget" native="true"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>170</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>170</height>
         </size>
        </property>
        <property name="title">
         <string>Motor Statistics</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutMotorDesignation">
           <item>
            <widget class="QLabel" name="labelMotorDesignationText">
             <property name="text">
              <string>Motor Designation:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelMotorDesignation">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutDeliveredISP">
           <item>
            <widget class="QLabel" name="labelDeliveredISPText">
             <property name="text">
              <string>Delivered ISP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDeliveredISP">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayoutPeakPressure">
           <item>
            <widget class="QLabel" name="labelPeakPressureText">
             <property name="text">
              <string>Peak Pressure:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPeakPressure">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayoutPeakKN">
           <item>
            <widget class="QLabel" name="labelPeakKNText">
             <property name="text">
              <string>Peak Kn:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPeakKN">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutBurnTime">
           <item>
            <widget class="QLabel" name="labelBurnTimeText">
             <property name="text">
              <string>Burn Time:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelBurnTime">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutImpulse">
           <item>
            <widget class="QLabel" name="labelImpulseText">
             <property name="text">
              <string>Impulse:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelImpulse">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QHBoxLayout" name="horizontalLayoutPropellantLength">
           <item>
            <widget class="QLabel" name="labelPropellantLengthText">
             <property name="text">
              <string>Propellant Length:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPropellantLength">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="2">
          <layout class="QHBoxLayout" name="horizontalLayoutPeakMassFlux">
           <item>
            <widget class="QLabel" name="labelPeakMassFluxText">
             <property name="text">
              <string>Peak Mass Flux:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPeakMassFlux">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <layout class="QHBoxLayout" name="horizontalLayoutPortThroatRatio">
           <item>
            <widget class="QLabel" name="labelPortThroatRatioText">
             <property name="text">
              <string>Port/Throat Ratio:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPortThroatRatio">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayoutInitialKN">
           <item>
            <widget class="QLabel" name="labelInitialKNText">
             <property name="text">
              <string>Initial Kn:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelInitialKN">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayoutAveragePressure">
           <item>
            <widget class="QLabel" name="labelAveragePressureText">
             <property name="text">
              <string>Average Pressure:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelAveragePressure">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayoutPropellantMass">
           <item>
            <widget class="QLabel" name="labelPropellantMassText">
             <property name="text">
              <string>Propellant Mass:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPropellantMass">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutVolumeLoading">
           <item>
            <widget class="QLabel" name="labelVolumeLoadingText">
             <property name="text">
              <string>Volume Loading:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelVolumeLoading">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayoutIdealThrustCoefficient">
           <item>
            <widget class="QLabel" name="labelIdealThrustCoefficientText">
             <property name="text">
              <string>Ideal Thrust Coefficient:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelIdealThrustCoefficient">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="2">
          <layout class="QHBoxLayout" name="horizontalLayoutDeliveredThrustCoefficient">
           <item>
            <widget class="QLabel" name="labelDeliveredThrustCoefficientText">
             <property name="text">
              <string>Delivered Thrust Coefficient:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDeliveredThrustCoefficient">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open Recent</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSimulate">
    <property name="title">
     <string>Simulate</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRunSimulation"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionPropellantEditor"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutOpenMotor"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSimulate"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRunSimulation">
   <property name="text">
    <string>Run Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAboutOpenMotor">
   <property name="text">
    <string>About openMotor</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPropellantEditor">
   <property name="text">
    <string>Propellant Editor</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResultsWidget</class>
   <extends>QWidget</extends>
   <header location="global">uilib.widgets.resultsWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MotorEditor</class>
   <extends>QWidget</extends>
   <header location="global">uilib.widgets.motorEditor</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
